q2_dada2/tests/data/sample_seqs_paired/MANIFEST

```diff
@@ -1,21 +1,23 @@
 sample-id,filename,direction
-BAQ3473.2,BAQ3473.2_2_L001_R1_001.fastq.gz,forward
-BAQ3473.2,BAQ3473.2_2_L001_R2_001.fastq.gz,reverse
-BAQ3473.3,BAQ3473.3_10_L001_R1_001.fastq.gz,forward
-BAQ3473.3,BAQ3473.3_10_L001_R2_001.fastq.gz,reverse
-BAQ4166.2,BAQ4166.2_21_L001_R1_001.fastq.gz,forward
-BAQ4166.2,BAQ4166.2_21_L001_R2_001.fastq.gz,reverse
-PAP130.2,PAP130.2_22_L001_R1_001.fastq.gz,forward
-PAP130.2,PAP130.2_22_L001_R2_001.fastq.gz,reverse
-YUN1242.1,YUN1242.1_31_L001_R1_001.fastq.gz,forward
-YUN1242.1,YUN1242.1_31_L001_R2_001.fastq.gz,reverse
-BAQ2420.2,BAQ2420.2_36_L001_R1_001.fastq.gz,forward
-BAQ2420.2,BAQ2420.2_36_L001_R2_001.fastq.gz,reverse
-YUN1242.3,YUN1242.3_44_L001_R1_001.fastq.gz,forward
-YUN1242.3,YUN1242.3_44_L001_R2_001.fastq.gz,reverse
-BAQ4166.3,BAQ4166.3_48_L001_R1_001.fastq.gz,forward
-BAQ4166.3,BAQ4166.3_48_L001_R2_001.fastq.gz,reverse
-YUN1005.3,YUN1005.3_49_L001_R1_001.fastq.gz,forward
-YUN1005.3,YUN1005.3_49_L001_R2_001.fastq.gz,reverse
-YUN3153.3,YUN3153.3_60_L001_R1_001.fastq.gz,forward
-YUN3153.3,YUN3153.3_60_L001_R2_001.fastq.gz,reverse
+BAQ3473.2,BAQ3473.2_0_L001_R1_001.fastq.gz,forward
```

Comment:
What happened with all of these files? It looks like maybe they all got renamed or something?

Reply:
Yeah they did, I tried adding the "Empty" sample by modifying the Manifest file, but they were still being ignored.  So I imported them into qiime with 2021.2, then exported them, which worked but it seems to have renamed a lot of the files.